package.json

```diff
@@ -22,7 +22,7 @@
     "cypress": "npx cypress open --component",
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
-    "parser": "peggy -o lib/parser.js --format es src/parser.peggy"
+    "parser": "peggy src/sympyParser/sympy.pegjs src/sympyParser/sympyLogic.pegjs src/sympyParser/sympyTerminals.pegjs && npx eslint 'src/sympyParser/sympy.js' --fix"
```

Comment:
Interesting, are you generating an AST using this library?

Reply:
kind of - it also allows rewrites per parse rule so the conversion happens as you parse in this case rather than exporting an explicit AST that you then re-compile into a new language